mobile/src/main/res/xml/preferences.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.preference.PreferenceScreen
```

Comment:
For consistency I'd like to see this in all pref xml files.

Reply:
Good point, I'll update that.